src/main/resources/application.yaml

```diff
@@ -125,6 +125,9 @@ azure:
     instrumentation-key: ${APPINSIGHTS_INSTRUMENTATIONKEY:dummy}
 
 features:
+  external-cookie-link:
```

Comment:
I like this style better than the `external-contact-page` one - can we make them consistent please.